tumblr-photo-video-ripper.py

```diff
@@ -196,12 +196,21 @@ def _download_media(self, site, medium_type, start):
                 print("Site %s does not exist" % site)
                 break
 
+            response_file = "{0}/{0}_{1}_{2}_{3}.response.xml".format(site, medium_type, MEDIA_NUM, start)
+            with open(response_file, "w") as text_file:
+                text_file.write(response.content)
+
             try:
                 xml_cleaned = re.sub(u'[^\x20-\x7f]+',
                                      u'', response.content.decode('utf-8'))
                 data = xmltodict.parse(xml_cleaned)
                 posts = data["tumblr"]["posts"]["post"]
                 for post in posts:
+                    post_json_file = "{0}/{0}_post_id_{1}.post.json".format(site, post['@id'])
```

Comment:
There will be plenty of json files here. Do you really want to do like this?

Reply:
You are right!
Even it's crazy comfortable for me to access each json file separately to check if it's synced with post correctly (rather then bulk xml file), most users may enjoy actually having less files and worry with extracting posts from xml afterwards.

Therefore, I propose introducing flag instead of removing this functionality.
However, I will absolutely understand if you have different judgement,
PTAL how it looks now. (Btw. I just tested after changes, seems to work with ==False and ==True)